src/components/Carousel.tsx

```diff
@@ -1,18 +1,122 @@
 import React from 'react';
 import './Carousel.scss';
 
-const Carousel: React.FC = () => (
-  <div className="Carousel">
-    <ul className="Carousel__list">
-      <li><img src="./img/1.png" alt="1" /></li>
-      <li><img src="./img/1.png" alt="2" /></li>
-      <li><img src="./img/1.png" alt="3" /></li>
-      <li><img src="./img/1.png" alt="4" /></li>
-    </ul>
-
-    <button type="button">Prev</button>
-    <button type="button">Next</button>
-  </div>
-);
+type Props = {
+  images: string[],
+  frameSize: number,
+  step: number,
+  animationDuration: number,
+  infinite: boolean,
+  itemWidth: number,
+};
+type State = {
+  itemOffset: number,
+};
+
+class Carousel extends React.Component<Props, State> {
+  state = {
+    itemOffset: 0,
+  };
+
+  clickNext = () => {
+    const {
+      itemWidth, images, infinite, frameSize,
+    } = this.props;
```

Comment:
Please fix all cases if present

```suggestion
    const {
      itemWidth,
      images,
      infinite,
      frameSize,
    } = this.props;
```